File: account/src/main/java/io/github/susimsek/account/AccountApplication.java
package io.github.susimsek.account;

import io.github.susimsek.account.dto.AccountContactInfoDTO;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@EnableDiscoveryClient
@EnableConfigurationProperties(value = AccountContactInfoDTO.class)
@SpringBootApplication
public class AccountApplication {

	public static void main(String[] args) {
		SpringApplication.run(AccountApplication.class, args);
	}

}

File: account/src/main/kotlin/io/github/susimsek/account/AccountApplication.kt
package io.github.susimsek.account

import io.github.susimsek.account.dto.AccountContactInfoDTO
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@EnableConfigurationProperties(value = [AccountContactInfoDTO::class])
@SpringBootApplication
class AccountApplication
fun main(args: Array<String>) {;
    runApplication<AccountApplication>(*args)
}
